### ISSUE TITLE
`sudo apt install ros-${ROS_DISTRO}-ament-cmake-clang-format` needed for building packages

### ISSUE THREAD

**Issue by ljaniec** [MOCAP4ROS2-Project/mocap4ros2_optitrack#25]
Hello,

I had to install this additional package before I could build the packages from this project. Perhaps it can be mentioned somewhere.
OS: Ubuntu 22.04, ROS 2 Humble.

Best regards,
Łukasz Janiec

**Comment by jmguerreroh** [comment]
Hi @ljaniec ,

Did you use ``rosdep`` to install all the dependencies? As you can check in package.xml, it is included at line 40  <test_depend>ament_cmake_clang_format</test_depend>

Anyway, we have updated the README to include this command:
```bash
rosdep install --from-paths src --ignore-src -r -y
```

Thank you for your feedback! :)

**Comment by ljaniec** [comment]
Yes, I didn't use rosdep, I had forgotten about it (everything worked before, so I didn't think about it). 

Thanks for the update with the command!
